manifests/integrations/activemq_xml.pp

```diff
@@ -42,49 +51,41 @@
 #       detailed_subscribers: ['subscriber1', 'subscriber2', 'subscriber3']
 #
 #
-class datadog_agent::integrations::activemq_xml(
-  String $url                                   = 'http://localhost:8161',
-  Boolean $supress_errors                       = false,
-  Optional[String] $username                    = undef,
-  Optional[String] $password                    = undef,
-  Optional[Array[String]] $detailed_queues      = [],
-  Optional[Array[String]] $detailed_topics      = [],
-  Optional[Array[String]] $detailed_subscribers = [],
-  Optional[Array] $instances                    = undef,
+class datadog_agent::integrations::activemq_xml (
+  String $url                         = 'http://localhost:8161',
+  Boolean $supress_errors             = false,
+  Optional[String] $username          = undef,
+  Optional[String] $password          = undef,
+  Array[String] $detailed_queues      = [],
+  Array[String] $detailed_topics      = [],
+  Array[String] $detailed_subscribers = [],
+  Optional[Array] $instances          = undef,
 ) inherits datadog_agent::params {
-  require ::datadog_agent
+  require datadog_agent
 
-  $legacy_dst = "${datadog_agent::params::legacy_conf_dir}/activemq_xml.yaml"
-  if $::datadog_agent::_agent_major_version > 5 {
-    $dst_dir = "${datadog_agent::params::conf_dir}/activemq_xml.d"
-    file { $legacy_dst:
-      ensure => 'absent'
-    }
+  $dst_dir = "${datadog_agent::params::conf_dir}/activemq_xml.d"
 
-    file { $dst_dir:
-      ensure  => directory,
-      owner   => $datadog_agent::dd_user,
-      group   => $datadog_agent::params::dd_group,
-      mode    => $datadog_agent::params::permissions_directory,
-      require => Package[$datadog_agent::params::package_name],
-      notify  => Service[$datadog_agent::params::service_name]
-    }
-    $dst = "${dst_dir}/conf.yaml"
-  } else {
-    $dst = $legacy_dst
+  file { $dst_dir:
+    ensure  => directory,
+    owner   => $datadog_agent::dd_user,
+    group   => $datadog_agent::params::dd_group,
+    mode    => $datadog_agent::params::permissions_directory,
+    require => Package[$datadog_agent::params::package_name],
+    notify  => Service[$datadog_agent::params::service_name],
   }
+  $dst = "${dst_dir}/conf.yaml"
 
   if !$instances and $url {
     $_instances = [{
-      'url'                  => $url,
-      'username'             => $username,
-      'password'             => $password,
-      'supress_errors'       => $supress_errors,
-      'detailed_queues'      => $detailed_queues,
-      'detailed_topics'      => $detailed_topics,
-      'detailed_subscribers' => $detailed_subscribers,
+        'url'                  => $url,
+        'username'             => $username,
+        'password'             => $password,
+        'supress_errors'       => $supress_errors,
```

Comment:
```suggestion
        'suppress_errors'       => $suppress_errors,
```

Reply:
This suggested change would be a breaking change since it's changing a config option. I'll add a corrected config option in a later release

Reply:
Fixed!